analysis_options.yaml

```diff
@@ -6,6 +6,11 @@ analyzer:
     strict-casts: true
     strict-inference: true
     strict-raw-types: true
+  exclude:
+    - 'example/src/example_test_golden.dart'
+    - 'test/src/goldens/test_library_file_no_part.dart'
+    - 'test/src/goldens/test_library_file_part_of.dart'
+    - 'test/src/goldens/test_library_file_part_of_current.dart'
```

Comment:
For some reason, just excluding `test/src/goldens` entirely leaves out some issues in `test/src/goldens/test_library_file_part_of_current.dart` on my machine (Dart 3.5.1), although it ignores other files there.
This could be a bug in the analyzer. I don't have time to investigate or strip down a minimal reproducible example.

Reply:
analyzer has had some issues lately w/ analysis_options and mono repos